css/reset.css

```diff
@@ -3,3 +3,22 @@
   padding: 0;
   list-style: none;
 }
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Spartan, sans-serif;
+  font-size: 32px;
+}
+
+body {
+  height: 100vh;
+  background-color: var(--bg-theme);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
```

Comment:
Why is this needed for the body element? Can we move this to div or main element?